resources/lang/pt-br/beatmaps.php

```diff
@@ -134,12 +159,20 @@
                 ],
                 'supporter-only' => 'Você precisa de uma supporter tag para acessar rankings de amigos e de país!',
                 'supporter-link' => 'Clique <a href=":link">aqui</a> para ver todas as novas funções às quais você ganha acesso!',
-                'global' => 'Ranking Global',
-                'country' => 'Ranking de País',
-                'friend' => 'Ranking de Amigos',
+                'global' => 'Ranking global',
+                'country' => 'Ranking de país',
+                'friend' => 'Ranking de amigos',
```

Comment:
for consistency you shouldn't change these capital characters. In english it's "Global Ranking" so we did put "Ranking Global". Same for the rest.